README.md

```diff
@@ -104,22 +104,21 @@ mfusepy requires FUSE 2.6 (or later) and runs on:
 - Mac OS X (Intel, PowerPC)
 - FreeBSD (i386, amd64)
 
-While FUSE is (at least in the Unix world) a [Kernel feature](https://man7.org/linux/man-pages/man4/fuse.4.html), several user space libraries exist for easy access.
+While FUSE is (at least in the Unix world) a [kernel feature](https://man7.org/linux/man-pages/man4/fuse.4.html), several user-space libraries exist for easy access.
 `libfuse` acts as the reference implementation.
 
  - [libfuse](https://github.com/libfuse/libfuse) (Linux, FreeBSD) (fuse.h [2](https://github.com/libfuse/libfuse/blob/fuse-2_9_bugfix/include/fuse.h) [3](https://github.com/libfuse/libfuse/blob/master/include/fuse.h))
  - [libfuse](https://github.com/openbsd/src/tree/master/lib/libfuse) (OpenBSD) (fuse.h [2](https://github.com/openbsd/src/blob/master/lib/libfuse/fuse.h))
  - [librefuse](https://github.com/NetBSD/src/tree/netbsd-8/lib/librefuse) (NetBSD) through [PUFFS](https://en.wikipedia.org/wiki/PUFFS_(NetBSD)) (fuse.h [2](https://github.com/NetBSD/src/blob/netbsd-8/lib/librefuse/fuse.h))
- - [FUSE for macOS](https://github.com/osxfuse/osxfuse) (OSX) (fuse.h [2](https://github.com/osxfuse/fuse/blob/master/include/fuse.h))
- - [MacFUSE](https://code.google.com/archive/p/macfuse/) (OSX), no longer maintained
- - [MacFUSE](https://macfuse.github.io/) (OSX), [Github](https://github.com/macfuse/library)
- - [FUSE-T](https://www.fuse-t.org/) (OSX), [Github](https://github.com/macos-fuse-t/fuse-t)
+ - [macFUSE](https://github.com/macfuse/macfuse) (macOS), previously called osxfuse, (fuse.h see [library links there](https://github.com/macfuse/macfuse))
```

Comment:
The links at the end were supposed to link to the exact fuse.h files and the link name (2 or 3) is the libfuse version. Now, the link is simply a duplicate of the main GitHub link.